### DISCUSSION TITLE
SMS courier configuration question

### DISCUSSION THREAD

**Discussion by renom** [ory/awesome-ory#96] [Q&A] [answered]
Screenshot from [configuration reference](https://www.ory.sh/docs/kratos/reference/configuration):

<img src="https://github.com/user-attachments/assets/dc06d5d6-4c21-4fb7-98b8-100d4669c79a" width="80%"/>

It seems there's two ways to configure SMS. Which one should I choose? (`courier/sms` vs `courier/channels`)

**Comment by aeneasr** [comment, marked as answer]
The channels one!
